common/app/experiments/ExperimentsDefinition.scala

```diff
@@ -5,7 +5,13 @@ import views.support.CamelCase
 
 class CanCheckExperiment(e: ExperimentsDefinition)
 
+sealed trait Participation
```

Comment:
I'm surprised we have to define this stuff here. It seems general enough to be pre-defined for all experiments - something to think about anyway (though not for this PR).

Reply:
👍